sqlalchemy_ibmi/base.py

```diff
@@ -807,6 +807,7 @@ def dbapi(cls):
         'database': ('DATABASE', str, None),
         'use_system_naming': ('NAM', to_bool, False),
         'trim_char_fields': ('TRIMCHAR', to_bool, None),
+        'lob_threshold_kb': ('maxfieldlen', str, None),
```

Comment:
```suggestion
        'lob_threshold_kb': ('MAXFIELDLEN', int, None),
```

ODBC keyword should be uppercase to work around the issue as noted in the comment above. Also, it should be set as an integer type for proper validation.